src/storage/src/monitor/hummock_state_store_metrics.rs

```diff
@@ -445,6 +449,31 @@ impl HummockStateStoreMetrics {
         let event_handler_latency =
             register_histogram_vec_with_registry!(opts, &["event_type"], registry).unwrap();
 
+        let safe_version_hit = GenericCounter::new(
+            "state_store_safe_version_hit",
+            "The total count of a safe version that can be retrieved successfully",
+        )
+        .unwrap();
+        registry
+            .register(Box::new(safe_version_hit.clone()))
+            .unwrap();
+
+        let safe_version_miss = GenericCounter::new(
+            "state_store_safe_version_miss",
+            "The total count of a safe version that cannot be retrieved",
+        )
+        .unwrap();
+        registry
+            .register(Box::new(safe_version_miss.clone()))
+            .unwrap();
+
+        let opts = histogram_opts!(
+            "safe_version_hit_index_histogram",
+            "The index of safe version found in recent versions",
+            exponential_buckets(1.0, 2.0, 10).unwrap(),
+        );
+        let safe_version_hit_index = register_histogram_with_registry!(opts, registry).unwrap();
```

Comment:
I think it's not that necessary to monitor the hit index. Hit and miss rate should be enough.

Reply:
Ok